contrib/metacontroller/UPDGRADE.md

```diff
@@ -0,0 +1,11 @@
+# Upgrade Metacontroller
+
+Metacontroller is pulled from [Kubeflow Pipelines third-party folder](https://github.com/kubeflow/pipelines/tree/master/manifests/kustomize/third-party/metacontroller). To update this component specify the desired KFP version and run the following command in console from the root directory **/**:
```

Comment:
Can we call root directory `../..` instead of `**/**` ? Or is there some convention to use `**` to represent upper folder?

Reply:
Fixed. Thank you!